.env.development

```diff
@@ -12,26 +12,13 @@ LOGOUT_URL='http://localhost:18000/logout'
 MARKETING_SITE_BASE_URL='http://localhost:18000'
 ORDER_HISTORY_URL='localhost:1996/orders'
 REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
-SEGMENT_KEY=null
```

Comment:
is this because 'null' is treated as actually a truthy value? we have had this issue in our other repoes too where we had to use '' vs anything else for False, and True respectively

Reply:
Yes. null gets cast to a string and becomes truthy. Same with false. An empty string is falsy, though, when it hits JS so makes a much better default.